test/exercism/homework_test.rb

```diff
@@ -0,0 +1,72 @@
+require_relative '../test_helper'
+require_relative '../active_record_helper'
+require_relative '../../lib/exercism'
+require_relative '../../lib/exercism/homework'
+
+class HomeworkTest < Minitest::Test
+  include DBCleaner
+
+  def alice
+    @alice ||= User.create(username: 'alice')
+  end
+
+  def teardown
+    super
+    @alice = nil
+    @language = nil
+    @skipped = nil
+    @fetched = nil
+    @recent = nil
+  end
+
+  def create_exercises
+    @language = 'ruby'
+    time = Time.now
+
+    @skipped = UserExercise.create(user: alice,
+                                   language: @language,
+                                   slug: 'leap',
+                                   skipped_at: time)
+
+    @fetched = UserExercise.create(user: alice,
+                                   language: @language,
+                                   slug: 'clock',
+                                   fetched_at: time)
+
+    @recent = UserExercise.create(user: alice,
```

Comment:
I know ruby folks do this thing where they break long lines... I really don't like it. I find that this ragged indentation makes the readability a lot worse.

There's no reason to store these in variables at all, really. It's just as easy to use hard-coded values in the assertions (and easier to read the assertions, as well).

I would do something like this, inline in the test:

```
now = Time.now.utc
attributes = {
  user: alice,
  language: 'ruby'
}
UserExercise.create(attributes.merge(slug: 'leap', skipped_at: now))
UserExercise.create(attributes.merge(slug: 'clock', fetched_at: now))
UserExercise.create(attributes.merge(slug: 'gigasecond', last_iteration_at: now))
```